lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3432,7 +3438,7 @@ def test_get_student_progress_url(self):
     def test_get_student_progress_url_from_uname(self):
         """ Test that progress_url is in the successful response. """
         url = reverse('get_student_progress_url', kwargs={'course_id': text_type(self.course.id)})
-        data = {'unique_student_identifier': self.students[0].username.encode("utf-8")}
+        data = {'unique_student_identifier': self.students[0].username}
```

Comment:
it added  `b` prefix like this with post data.
```b'robot'``` prefix with username and when it reaches inside view it returns some int value and user lookup db query fails.